04-lection2/01-histogram/index.html

```diff
@@ -6,6 +6,62 @@
         <link rel="stylesheet" href="./histogram.css">
     </head>
     <body>
+<div class="histogram">
+<div class="histogram__info">
+    <div class="info__total total">
+        <div class="total__title">Total orders</div>
+        <div class="total__value">50254</div>
+    </div>
+    <div class="histogram__link">
+        <a href="#">View all</a>
+    </div>
```

Comment:
Выглядит как ненужная обертка)
```suggestion
<a class="histogram__link" href="#">View all</a>
```

```suggestion
.histogram__link {
    display: block;
    color: #4263EB;
    text-decoration: none; // чтобы убрать подчеркивание у ссылки
}
```